cloud/main.js

```diff
@@ -1,5 +1,3 @@
-var Parse = require('parse/node').Parse;
```

Comment:
Needed to remove that import because of parse-cloud-express and conflicting dependencies.